.github/workflows/master_only.yml

```diff
@@ -52,10 +53,10 @@ jobs:
             docker push gcr.io/kf-feast/feast-${{ matrix.component }}:${VERSION_WITHOUT_PREFIX}
 
             # Also update "latest" image if tagged commit is pushed to stable branch
-            HIGHEST_SEMVER_TAG=$(git tag -l --sort -version:refname | head -n 1)
+            HIGHEST_SEMVER_TAG=$(get_tag_release -m)
             echo "Only push to latest tag if tag is the highest semver version $HIGHEST_SEMVER_TAG"
 
-            if [ "${VERSION_WITHOUT_PREFIX}" == "${HIGHEST_SEMVER_TAG:1}" ]
+            if [ "${VERSION_WITHOUT_PREFIX}" = "${HIGHEST_SEMVER_TAG:1}" ]
```

Comment:
maybe `[[ "${VERSION_WITHOUT_PREFIX}" == "${HIGHEST_SEMVER_TAG:1}" ]]`

Reply:
`[]` implies the `test` command. According to its [manpage](https://linux.die.net/man/1/test), it uses single `=` for string comparision.